pointers/pointers.c

```diff
@@ -28,25 +34,85 @@ void string_copy(char *x, char *y)
 */
 char *find_char(char *str, int c)
 {
+    // int sLen = string_length(str);
 
+    // for (int i = 0; i <= sLen; i++)
+    // {
+    //     if (str[i] == c)
+    //     {
+    //         return &str[i];
+    //     }
+    // }
+
+    // Bit more concise. Doesn't need a full for loop implementation, since we can just use the null terminator to stop the loop at worst. No pesky worrying about off by 1 errors.
+    while (*str != 0)
```

Comment:
very nice! my old solution works _exactly_ the same way

Reply:
👍